### ISSUE TITLE
Won't share items

### ISSUE THREAD

**Issue by Joel0420** [jmausolf/poshmark_sharing#14]
My poshwar bot is running fine but for some reason it is "sharing 0 items in closet". Is there a parameter or line of code that I have to change telling it how many active items I have in my closet. 7 scrolls(the default) should be enough to at least get through most of my items. It seems like it tries to open my closet and then the browser shuts down.
![Screen Shot 2019-04-18 at 10 42 34 PM](https://user-images.githubusercontent.com/45363445/56402482-6b033980-622b-11e9-991d-0351e55a4c7a.png)

**Comment by nicolealaine** [comment]
Hi there, looks like you might have your email address where your closet name should be. Try updating your credentials.py to include your closet name rather than your login email.

**Comment by jmausolf** [comment]
@Joel0420, I would agree with @nicolealaine. Your credentials file should have your poshmark username (closetname). See the [README](https://github.com/jmausolf/poshmark_sharing/blob/master/README.md). The code uses the closet name in the credentials.py file (or the passed alternate account name) and navigates to that page using the following function:

```python
def get_seller_page_url(poshmark_account):
    url_stem = 'https://poshmark.com/closet/'
    available = '?availability=available'
    url = '{}{}{}'.format(url_stem, poshmark_account, available)
    return url
```
In other words, it is trying to share items at an invalid closet: [https://poshmark.com/closet/joel.jerez0420@gmail.com?availability=available](https://poshmark.com/closet/joel.jerez0420@gmail.com?availability=available), which explains why 0 items are found.

For example, I get the same results if I try to share items for your email address (not your user name). 

![image](https://user-images.githubusercontent.com/10625806/56551417-f7746b80-654d-11e9-9581-592e88ab4fde.png)

The issue does suggest a possible feature improvement, such as a regex function to detect when either the username (or alternate account) is an email instead of a closetname, and in such as case raise a warning and exit.

**Comment by nstepka** [comment]
Hi ya'all!  I just finished creating a java/selenium bot that will send an offer to all likers.  Feel free to check it out/improve on it

https://github.com/nstepka/PoshMarkDiscountBot
